draft-ietf-acme-acme.md

```diff
@@ -398,7 +399,7 @@ private key unless otherwise specified.  The server MUST verify the JWS before
 processing the request. Encapsulating request bodies in JWS provides
 authentication of requests.
 
-JWS objects sent in ACME requests MUST meet the following additional criteria:
+JWS objects sent as the body of ACME requests MUST meet the following additional criteria:
```

Comment:
This doesn't quite parse.  How about we change to the singular: "A JWS object sent as the body of an ACME request MUST..."